### PR TITLE
Add more MongoDB metrics

### PR DESCRIPTION
**Pull request by bozaro**
- flushes_total_time_ns - Transaction checkpoint total time
- wtcache_pages_read_info - Pages read into cache
- wtcache_pages_requested_from - Pages requested from the cache
- wtcache_internal_pages_evicted - Internal pages evicted
- wtcache_modified_pages_evicted - Modified pages evicted
- wtcache_unmodified_pages_evicted - Unmodified pages evicted

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/mongodb/mongostat.go [hunk 1]
```diff
@@ -681,8 +692,19 @@ func NewStatLine(oldMongo, newMongo MongoStatus, key string, all bool, sampleSec
...
+		returnVal.PagesReadIntoCache = newStat.WiredTiger.Cache.PagesReadIntoCache
+		returnVal.PagesRequestedFromCache = newStat.WiredTiger.Cache.PagesRequestedFromCache
 		returnVal.ServerEvictingPages = newStat.WiredTiger.Cache.ServerEvictingPages
 		returnVal.WorkerThreadEvictingPages = newStat.WiredTiger.Cache.WorkerThreadEvictingPages
+
+		returnVal.InternalPagesEvicted = newStat.WiredTiger.Cache.InternalPagesEvicted
+		returnVal.ModifiedPagesEvicted = newStat.WiredTiger.Cache.ModifiedPagesEvicted
+		returnVal.UnmodifiedPagesEvicted = newStat.WiredTiger.Cache.UnmodifiedPagesEvicted
+
+		returnVal.FlushesTotalTime = float64(newStat.WiredTiger.Transaction.TransCheckpointsTotalTimeMsecs) / float64(time.Millisecond)
```
Comment (danielnelson) [inline plugins/inputs/mongodb/mongostat.go]: Our latest style for durations is to report as an integer in nanoseconds: TotalTimeMsecs * time.Millisecond

Comment (bozaro) [reply plugins/inputs/mongodb/mongostat.go]: I can't find any exists metrics in nanoseconds.
In `inputs/cpu/cpu.go` time reported as `float64` in `seconds`.
I don't known correct type for `FlushesToTotalTime`: `float64`, `int64`, `time.Duration`, `internal.Duration` or something else...

Comment (danielnelson) [reply plugins/inputs/mongodb/mongostat.go]: The time and duration values are not nearly as uniform as I would like, though I can think of several that are using ns currently (zipkin, internal, kapacitor).  In the future I would like to normalize these time datatypes more, perhaps even adding them as first class types in Telegraf.

Use int64 for the time in ns.

**Comment by bozaro** [comment]
Rebase to master and update `plugins/inputs/mongodb/README.md`

**Comment by bozaro** [comment]
Thanks a lot.

**Pull request merged**